<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>788</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="Plotter" name="widgetPlotter" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>560</width>
      <height>561</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>471</width>
      <height>431</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPV">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;PutVoxel&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>PV</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCV">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>CV</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPB">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>PB</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCB">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>CB</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPS">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>PS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCS">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>CS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPE">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>PE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCE">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>20</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>CE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonXY">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>20</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>XY</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonYZ">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>20</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>YZ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonXZ">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>20</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>XZ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOFF">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>20</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>OFF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSV">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>20</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>SV</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRotClk">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>80</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>RotClk</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRotCClk">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>80</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>RotCClk</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonVECT">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>20</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>VECT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonInv">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>80</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Inv</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumberX">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>660</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumberY">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>660</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumberLinha">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>660</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumberColuna">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>660</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumberPlano">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>660</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonNewSz">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>80</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>New</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>140</y>
      <width>401</width>
      <height>568</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_8">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Box Size</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>X</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSlider" name="horizontalSliderX">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLCDNumber" name="lcdNumber"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Y</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSlider" name="horizontalSliderY">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLCDNumber" name="lcdNumber_2"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Z</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSlider" name="horizontalSliderZ">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLCDNumber" name="lcdNumber_3"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Sphere's Radius</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Rd</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSlider" name="horizontalSliderRD">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLCDNumber" name="lcdNumber_4"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Ellipsoid's Radius</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>RdX</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSlider" name="horizontalSliderRX">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLCDNumber" name="lcdNumber_5"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>RdY</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSlider" name="horizontalSliderRY">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLCDNumber" name="lcdNumber_6"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>RdZ</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSlider" name="horizontalSliderRZ">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLCDNumber" name="lcdNumber_7"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Slice</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Pln</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSlider" name="horizontalSliderSlice">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLCDNumber" name="lcdNumber_8"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout_7" columnstretch="30,70">
       <item row="0" column="0">
        <widget class="ColorWindow" name="widgetColor" native="true"/>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Color</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>R</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSlider" name="horizontalSliderRed">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLCDNumber" name="lcdNumber_9"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>G</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSlider" name="horizontalSliderGreen">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLCDNumber" name="lcdNumber_10"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>B</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSlider" name="horizontalSliderBlue">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLCDNumber" name="lcdNumber_11"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>A</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSlider" name="horizontalSliderAlpha">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLCDNumber" name="lcdNumber_12"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionputVoxel"/>
   <addaction name="actioncutVoxel"/>
   <addaction name="actionputBox"/>
   <addaction name="actioncutBox"/>
   <addaction name="actionputSphere"/>
   <addaction name="actioncutSphere"/>
   <addaction name="actionputEllipsoid"/>
   <addaction name="actioncutEllipsoid"/>
   <addaction name="actionRotClk"/>
   <addaction name="actionRotCClk"/>
   <addaction name="actionInvert"/>
   <addaction name="actionNewBox"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionputVoxel">
   <property name="text">
    <string>putVoxel</string>
   </property>
  </action>
  <action name="actioncutVoxel">
   <property name="text">
    <string>cutVoxel</string>
   </property>
  </action>
  <action name="actionputBox">
   <property name="text">
    <string>putBox</string>
   </property>
  </action>
  <action name="actioncutBox">
   <property name="text">
    <string>cutBox</string>
   </property>
  </action>
  <action name="actionputSphere">
   <property name="text">
    <string>putSphere</string>
   </property>
  </action>
  <action name="actioncutSphere">
   <property name="text">
    <string>cutSphere</string>
   </property>
  </action>
  <action name="actionputEllipsoid">
   <property name="text">
    <string>putEllipsoid</string>
   </property>
  </action>
  <action name="actioncutEllipsoid">
   <property name="text">
    <string>cutEllipsoid</string>
   </property>
  </action>
  <action name="actionRotClk">
   <property name="text">
    <string>RotClk</string>
   </property>
  </action>
  <action name="actionRotCClk">
   <property name="text">
    <string>RotCClk</string>
   </property>
  </action>
  <action name="actionInvert">
   <property name="text">
    <string>Invert</string>
   </property>
  </action>
  <action name="actionNewBox">
   <property name="text">
    <string>NewBox</string>
   </property>
  </action>
  <action name="actionShow_OFF">
   <property name="text">
    <string>Show_OFF</string>
   </property>
  </action>
  <action name="actionShow_VECT">
   <property name="text">
    <string>Show_VECT</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Plotter</class>
   <extends>QWidget</extends>
   <header>plotter.h</header>
   <container>1</container>
   <slots>
    <signal>moveX(int)</signal>
    <signal>moveY(int)</signal>
    <signal>planeChosen(int)</signal>
    <slot>changeRed(int)</slot>
    <slot>changeGreen(int)</slot>
    <slot>changeBlue(int)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>ColorWindow</class>
   <extends>QWidget</extends>
   <header>colorwindow.h</header>
   <container>1</container>
   <slots>
    <slot>changeRed()</slot>
    <slot>changeGreen()</slot>
    <slot>changeBlue()</slot>
    <slot>changeAlpha()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSliderX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>809</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel">
     <x>966</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_2</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>807</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>966</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderZ</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_3</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>801</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>966</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderRD</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_4</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>815</x>
     <y>377</y>
    </hint>
    <hint type="destinationlabel">
     <x>966</x>
     <y>381</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderRX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_5</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>822</x>
     <y>450</y>
    </hint>
    <hint type="destinationlabel">
     <x>966</x>
     <y>454</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderRY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_6</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>828</x>
     <y>479</y>
    </hint>
    <hint type="destinationlabel">
     <x>966</x>
     <y>483</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderRZ</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_7</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>826</x>
     <y>508</y>
    </hint>
    <hint type="destinationlabel">
     <x>966</x>
     <y>512</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderSlice</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_8</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>896</x>
     <y>581</y>
    </hint>
    <hint type="destinationlabel">
     <x>966</x>
     <y>585</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderRed</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_9</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>896</x>
     <y>656</y>
    </hint>
    <hint type="destinationlabel">
     <x>966</x>
     <y>660</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderGreen</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_10</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>896</x>
     <y>685</y>
    </hint>
    <hint type="destinationlabel">
     <x>966</x>
     <y>689</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderBlue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_11</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>896</x>
     <y>714</y>
    </hint>
    <hint type="destinationlabel">
     <x>966</x>
     <y>718</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderAlpha</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_12</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>896</x>
     <y>743</y>
    </hint>
    <hint type="destinationlabel">
     <x>966</x>
     <y>747</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>widgetPlotter</sender>
   <signal>planeChosen(int)</signal>
   <receiver>lcdNumberPlano</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>521</x>
     <y>616</y>
    </hint>
    <hint type="destinationlabel">
     <x>537</x>
     <y>718</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderRed</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widgetPlotter</receiver>
   <slot>changeRed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>757</x>
     <y>652</y>
    </hint>
    <hint type="destinationlabel">
     <x>559</x>
     <y>645</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderGreen</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widgetPlotter</receiver>
   <slot>changeGreen(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>743</x>
     <y>682</y>
    </hint>
    <hint type="destinationlabel">
     <x>568</x>
     <y>664</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderBlue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widgetPlotter</receiver>
   <slot>changeBlue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>758</x>
     <y>704</y>
    </hint>
    <hint type="destinationlabel">
     <x>553</x>
     <y>677</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
